client/src/components/molecules/AccountStatus/Popover.module.scss

```diff
@@ -28,7 +28,7 @@ $popoverWidth: 18rem;
 }
 
 .popoverInfoline {
-    border-bottom: .05rem solid $brand-grey;
+    border-bottom: 0.05rem solid $brand-grey;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)